docs/data/material/components/menus/GroupedMenu.js

```diff
@@ -33,13 +33,32 @@ export default function GroupedMenu() {
         slotProps={{
           list: {
             'aria-labelledby': 'basic-button',
+            sx: {
+              py: 0,
```

Comment:
without this style, demo is looking bit weird in dark mode.

with style
<img width="75" alt="image" src="https://github.com/user-attachments/assets/85f6c112-abc5-45d5-92b0-e4d781e62e42" />

without style
<img width="80" alt="image" src="https://github.com/user-attachments/assets/bbf371cd-1e31-4a0e-afae-d56b0e1d3aeb" />